docs/trestle_author.md

```diff
@@ -409,6 +409,44 @@ Running `trestle author docs validate -tn docs_task -gh="Governed section"` will
 - If `--template-version 1.0.0` (`-tv`) is passed the header field `x-trestle-template-version` will be ignored and document will be forcefully validated against template of version `1.0.0`.
   Use this for testing purposes _only_ when you need to validate the document against a specific template. By default the template version will be determined based on `x-trestle-template-version` in the document.
 
+### Validating the documents against different templates
+
+Validation against multiple templates as stated before can be done, but there is another scenario that you can leverage on trestle to have multiple documents in the task folder corresponding to a 1 template.
+
+Please, take into consideration that for the validation to happen you will need to provide each instance of the template with a field called `x-trestle-template-type: insert_template_type_here` in the yaml header matching with the template name. With that, you will be able to create more than 1 instance document per template and give the instance the name desired.
+
```

Comment:
"... provide in each instance document in the task folder a field ..."
"... give the instance the desired name."

Reply:
done